.gitignore

```diff
@@ -22,4 +24,14 @@ VERSION
 endpoints.json
 generate-doc
 # Ignore vi backup files
-*~
\ No newline at end of file
+*~
+# Ignore patches.
+*.patch
```

Comment:
Well, I use `git format-patch` often and `git status` helps me find those patch files. That's just my personal preference. This is okay